pkg/cache/cache.go

```diff
@@ -194,48 +214,111 @@ type ByObject struct {
 	UnsafeDisableDeepCopy *bool
 }
 
+// Config describes all potential options for a given watch.
```

Comment:
well, informer maps 1:1 to a watch, so either that or watch would work but I think odds are higher that ppl know what a watch is than that they know what an informer is.

Cache is IMHO not good, because it encompasses the whole thing with potentially many watches/informers.